src/databricks/labs/remorph/coverage/commons.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=all
+# pylint: disable=too-many-instance-attributes,broad-exception-caught
```

Comment:
ignoring since this is for integration test

Reply:
@vijaypavann-db revert this

Reply:
Done Sundar